etc/sdk.api.md

```diff
@@ -217,6 +217,15 @@ export interface ListDirectoryResponse {
     items: DirectoryEntry[];
 }
 
+// @public (undocumented)
+export interface MakeFilePublicRequest {
+    // (undocumented)
+    bucket: string;
+    // (undocumented)
+    path: string;

```

Comment:
This was an array in the daemon, but I don't think we need multiple files in one call right?